modules/nw-ptp-introduction.adoc

```diff
@@ -0,0 +1,27 @@
+// Module included in the following assemblies:
+//
+// * networking/multiple_networks/configuring-ptp.adoc
+
+[id="ptp-introduction_{context}"]
+= About PTP
+
+The Precision Time Protocol (PTP) is used to synchronize clocks in a network. When used in conjunction with hardware support, PTP is capable of sub-microsecond accuracy, and is more accurate than Network Time Protocol (NTP).
+
+The `linuxptp` package includes the `ptp4l` and `phc2sys` programs for clock synchronization. `ptp4l` implements the PTP boundary clock and ordinary clock. `ptp4l` synchronizes the PTP hardware clock to the source clock with hardware time stamping and synchronizes the system clock to the source clock with software time stamping. `phc2sys` is used for hardware time stamping to synchronize the system clock to the PTP hardware clock on the network interface card (NIC).
+
+[id="ptp-elements_{context}"]
+== Elements of a PTP domain
+
+PTP is used to synconize multiple nodes connected in a network, with clocks for each node. The following type of clocks can be included in configurations:
+
+Grandmaster clock:: The grandmaster clock provides standard time information to other clocks across the network and ensures accurate and stable synchronisation. The grandmaster clock writes time stamps and responds to time requests from other clocks.
+
+Ordinary clock:: The ordinary clock has a single port connection that can play the role of source or destination clock, depending on its position in the network. The ordinary clock can read and write time stamps.
+
+Boundary clock:: The boundary clock has ports in two or more communication paths and can be a source and a destination to other destination clocks at the same time. The boundary clock works as a destination clock upstream. The destination clock receives the timing message, adjusts for delay, and then creates a new source time signal to pass down the network. The boundary clock produces a new timing packet that is still correctly synced with the source clock, and can reduce the number of connected devices reporting directly to the source clock. 
```

Comment:
Incorrect comma usage: The boundary clock produces a new timing packet that is still correctly synced with the source clock, <--(here) and can reduce the number of connected devices reporting directly to the source clock. 

You only need a comma before a coordinating conjunction if you're joining two independent clauses. You can just omit the comma and leave the and. Or you could use "it" as the subject and leave the comma, so ", and it can reduce the number..."